.travis.yml

```diff
@@ -37,3 +37,11 @@ after_success:
   - bash <(curl -s https://codecov.io/bash)
  # - bash scripts/pr_deploy.sh
  # - bash kubernetes/travis/deploy.sh
+
+deploy:
```

Comment:
Specify which directory to deploy to gh-pages.

Reply:
Added